<?xml version="1.0" encoding="UTF-8"?><testrun name="CoffeeMaker" project="CoffeeMaker" tests="11" started="11" failures="2" errors="0" ignored="0">
  <testsuite name="edu.towson.cis.cosc603.project5.coffeemaker.CoffeeMakerTest" time="0.052">
    <testcase name="testAddRecipe1" classname="edu.towson.cis.cosc603.project5.coffeemaker.CoffeeMakerTest" time="0.001"/>
    <testcase name="testAddRecipe2" classname="edu.towson.cis.cosc603.project5.coffeemaker.CoffeeMakerTest" time="0.0"/>
    <testcase name="testAddRecipe3" classname="edu.towson.cis.cosc603.project5.coffeemaker.CoffeeMakerTest" time="0.0"/>
    <testcase name="testAddRecipe4" classname="edu.towson.cis.cosc603.project5.coffeemaker.CoffeeMakerTest" time="0.006">
      <failure>junit.framework.AssertionFailedError&#13;
	at junit.framework.Assert.fail(Assert.java:55)&#13;
	at junit.framework.Assert.assertTrue(Assert.java:22)&#13;
	at junit.framework.Assert.assertFalse(Assert.java:39)&#13;
	at junit.framework.Assert.assertFalse(Assert.java:47)&#13;
	at junit.framework.TestCase.assertFalse(TestCase.java:219)&#13;
	at edu.towson.cis.cosc603.project5.coffeemaker.CoffeeMakerTest.testAddRecipe4(CoffeeMakerTest.java:68)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testEditRecipe1" classname="edu.towson.cis.cosc603.project5.coffeemaker.CoffeeMakerTest" time="0.001"/>
    <testcase name="testEditRecipe2" classname="edu.towson.cis.cosc603.project5.coffeemaker.CoffeeMakerTest" time="0.001"/>
    <testcase name="testDeleteRecipe1" classname="edu.towson.cis.cosc603.project5.coffeemaker.CoffeeMakerTest" time="0.001"/>
    <testcase name="testDeleteRecipe2" classname="edu.towson.cis.cosc603.project5.coffeemaker.CoffeeMakerTest" time="0.002"/>
    <testcase name="testCheckInventory1" classname="edu.towson.cis.cosc603.project5.coffeemaker.CoffeeMakerTest" time="0.032">
      <failure>junit.framework.AssertionFailedError: expected:&lt;10&gt; but was:&lt;15&gt;&#13;
	at junit.framework.Assert.fail(Assert.java:57)&#13;
	at junit.framework.Assert.failNotEquals(Assert.java:329)&#13;
	at junit.framework.Assert.assertEquals(Assert.java:78)&#13;
	at junit.framework.Assert.assertEquals(Assert.java:234)&#13;
	at junit.framework.Assert.assertEquals(Assert.java:241)&#13;
	at junit.framework.TestCase.assertEquals(TestCase.java:409)&#13;
	at edu.towson.cis.cosc603.project5.coffeemaker.CoffeeMakerTest.testCheckInventory1(CoffeeMakerTest.java:108)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testAddInventory1" classname="edu.towson.cis.cosc603.project5.coffeemaker.CoffeeMakerTest" time="0.001"/>
    <testcase name="testAddInventory2" classname="edu.towson.cis.cosc603.project5.coffeemaker.CoffeeMakerTest" time="0.004"/>
  </testsuite>
</testrun>
